src/contracts/libraries/Merkle.sol

```diff
@@ -164,4 +163,45 @@ library Merkle {
         //the first node in the layer is the root
         return layer[0];
     }
+
+    /**
+     * @notice this function returns the merkle root of a tree created from a set of leaves using keccak256 as its hash function
+     *  @param leaves the leaves of the merkle tree
+     *  @return The computed Merkle root of the tree.
+     *  @dev This pads to the next power of 2. very inefficient! just for POC
+     */
+    function merkleizeKeccak256(bytes32[] memory leaves) internal pure returns (bytes32) {
```

Comment:
Bitshifts are a bit cheaper than multiplying or dividing by 2 at the cost of readability.
```
100 >> 1
└ Decimal: 50

100 << 1
└ Decimal: 200
```